build.gradle

```diff
@@ -164,37 +206,93 @@ repositories {
 
-  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: '6.0.0'
+  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: versions.logging
 
   implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: log4JVersion
   implementation group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: log4JVersion
 
+  implementation group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
+  implementation group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
+  implementation group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
+  implementation group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
+
+  //implementation group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '4.0.0'
+
   testImplementation(platform('org.junit:junit-bom:5.8.2'))
   testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
   testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', {
     exclude group: 'junit', module: 'junit'
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
+  testImplementation "org.junit.vintage:junit-vintage-engine:${versions.junit}"
+
+  testImplementation group: 'org.pitest', name: 'pitest', version: versions.pitest
+  testImplementation 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.5.2'
+  testImplementation 'org.codehaus.sonar-plugins:sonar-pitest-plugin:0.5'
+
+  functionalTestImplementation sourceSets.main.runtimeClasspath
+  functionalTestImplementation sourceSets.test.runtimeClasspath
+
+  smokeTestImplementation sourceSets.main.runtimeClasspath
+  smokeTestImplementation sourceSets.test.runtimeClasspath
+
+  constraints {
+    implementation(group: 'com.jhlabs', name: 'filters', version: '2.0.235-1') {
+      because 'CVE-2005-0406'
+    }
+    implementation(group: 'com.ibm.icu', name: 'icu4j', version: '70.1') {
+      because 'CVE-2020-21913'
+    }
+    implementation(group: 'org.apache.pdfbox', name: 'pdfbox', version: '2.0.25') {
+      because 'CVE-2018-11797,CVE-2018-8036,CVE-2021-27807,CVE-2021-27906,CVE-2021-31811,CVE-2021-31812'
+    }
+    implementation(group: 'org.apache.pdfbox', name: 'xmpbox', version: '2.0.25') {
+      because 'CVE-2018-11797, CVE-2018-8036, CVE-2021-27807, CVE-2021-27906, CVE-2021-31811, CVE-2021-31812'
+    }
+    implementation(group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.64') {
+      because 'CVE-2020-15522'
+    }
+  }
+
 }
 
-mainClassName = 'uk.gov.hmcts.reform.demo.Application'
+mainClassName = 'uk.gov.hmcts.reform.sscs.Application'
 
 bootJar {
-  archiveFileName = "spring-boot-template.jar"
+  archiveFileName = "sscs-hearings-api.jar"
 
   manifest {
     attributes('Implementation-Version': project.version.toString())
```

Comment:
Change to
attributes('Implementation-Title': project.name, 'Implementation-Version': project.version.toString())

Reply:
Done